test/Microsoft.Azure.Devices.Edge.Test/helpers/Context.cs

```diff
@@ -122,8 +125,14 @@ string CheckDeviceIdLength(string deviceId)
 
         public string ConnectionString { get; }
 
+        public Dictionary<string, EdgeDevice> DeleteList { get; } = new Dictionary<string, EdgeDevice>();
```

Comment:
Yes it would, except `DeleteList` is special. It's the first (and only) instance variable in `Context` that doesn't come from the `context.json` file. It is state maintained during the tests, and it tells us which device identities to delete at the end. I kept it separate from all the variables whose values are parsed out of context.json in the constructor.

Reply:
Oh I see.